### ISSUE TITLE
Checkout Error - Woocommerce stripe connect plugin giving "Invalid Product" error on variable products

### ISSUE THREAD

**Issue by sathapa** [woocommerce/woocommerce#22229]
**"Invalid Product" for variable products on woocommerce stripe connect add-on**
There's an "Invalid Product" error that shows when one uses credit card payment via Stripe connect from wordpress plugin when the product is a variable product. I checked with the default theme, it still shows the same issue. 

If I replace the if condition on stripe-connect.php core file to just one liner, the error doesn't seem to show up.

`if ( 0 != $value->get_data()['variation_id'] ) {
						$price_id = $value->get_data()['variation_id'];
					} else {
						$price_id = $value->get_data()['product_id'];
					}
					$product        = new WC_Product( $price_id );`

to 

`$price_id = $value->get_data()['product_id'];
  $product        = new WC_Product( $price_id );`

**To Reproduce**
1. Go to 'Products'. Add new with variable product as type.
2. Got to website and add that product and checkout.
3. Checkout via Stripe connect and put down the credit card details. 
4. On submit, you'll see the "Invalid Product" error.

**Screenshots**
![screen shot 2018-12-14 at 12 02 23 pm](https://user-images.githubusercontent.com/28758613/50019672-26c00e00-ff99-11e8-8147-c3391465a9ba.png)
![screen shot 2018-12-14 at 12 02 59 pm](https://user-images.githubusercontent.com/28758613/50019681-2c1d5880-ff99-11e8-9792-33c6c093c8b0.png)

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
wp_version = '5.0.1';
db_version = 43764;
tinymce_version = '4800-20180716';
php_version = '5.2.4';
mysql_version = '5.0';
</details>

**Comment by rodrigoprimo** [comment]
Closing as this issue tracker is used exclusively for WooCommerce core issues. I'm not sure which Stripe extension you are using. The WooCommerce Stripe extension (https://woocommerce.com/products/stripe/) developed by Automattic does not have a `stripe-connect.php` file. Maybe you are using https://github.com/jason-engage/wordpress-stripe-connect? I found this repository on a quick search for `stripe-connect.php`. I'm not sure if this plugin is still maintained.
